### PR TITLE
[CTOR-571] Create Symbol WiNG SNMP doc page

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

**Review by lucie-dubrunfaut — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-symbol-wing-snmp.md [hunk 1]
```diff
@@ -0,0 +1,552 @@
...
+| Systems | Net-Symbol-Wing-Systems-SNMP-custom | Contrôle les systèmes |
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Net-Symbol-Wing-SNMP-custom** est utilisé.
+
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
+
+| Alias                | Modèle de service                                | Description                                                                        | Découverte |
+|:---------------------|:-------------------------------------------------|:-----------------------------------------------------------------------------------|:----------:|
+| Packet-Errors        | Net-Symbol-wing-Packet-Errors-Global-SNMP-custom | Contrôle le pourcentage de paquets en erreur/écarté de plusieurs interfaces réseau | X          |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-symbol-wing-snmp.md:33]:
```suggestion
| Packet-Errors        | Net-Symbol-wing-Packet-Errors-Global-SNMP-custom | Contrôle le pourcentage de paquets en erreur/écartés de plusieurs interfaces réseau | X          |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-symbol-wing-snmp.md [hunk 2]
```diff
@@ -0,0 +1,552 @@
...
+|:----------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Discover your resources through an SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule and create a template mapper for the **Net-Symbol-Wing-SNMP-custom** host template |
+
+Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/hosts-discovery) pour en savoir plus sur la découverte automatique d'hôtes.
+
+#### Découverte de services
+
+| Nom de la règle                         | Description                                                             |
+|:----------------------------------------|:------------------------------------------------------------------------|
+| Net-Symbol-Wing-SNMP-Packet-Errors-Name | Découvre les interfaces réseau et supervise les paquets erronés et rejetés  |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-symbol-wing-snmp.md:59]: signale au lieu de supervise?

File: pp/integrations/plugin-packs/procedures/network-symbol-wing-snmp.md [hunk 3]
```diff
@@ -0,0 +1,551 @@
...
+| Systems       | Net-Symbol-Wing-Systems-SNMP-custom | Check systems       |
+
+> The services listed above are created automatically when the **Net-Symbol-Wing-SNMP-custom** host template is used.
+
+</TabItem>
+<TabItem value="Not attached to a host template" label="Not attached to a host template">
+
+| Service Alias        | Service Template                                 | Service Description                                             | Discovery  |
+|:---------------------|:-------------------------------------------------|:----------------------------------------------------------------|:----------:|
+| Packet-Errors        | Net-Symbol-wing-Packet-Errors-Global-SNMP-custom | Check packets on errors/discards of multiple network interfaces | X          |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-symbol-wing-snmp.md:32]:
```suggestion
| Packet-Errors        | Net-Symbol-wing-Packet-Errors-Global-SNMP-custom | Check packets in error/discarded from multiple network interfaces | X          |
```

File: pp/integrations/plugin-packs/procedures/network-symbol-wing-snmp.md [hunk 4]
```diff
@@ -0,0 +1,551 @@
...
+| Traffic-Generic-Id   | Net-Symbol-Wing-Traffic-Generic-Id-SNMP-custom   | Check traffic of an network interface                           |            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-symbol-wing-snmp.md:33]:
```suggestion
| Traffic-Generic-Id   | Net-Symbol-Wing-Traffic-Generic-Id-SNMP-custom   | Check traffic of a network interface                           |            |
```

File: pp/integrations/plugin-packs/procedures/network-symbol-wing-snmp.md [hunk 5]
```diff
@@ -0,0 +1,551 @@
...
+| Traffic-Generic-Name | Net-Symbol-Wing-Traffic-Generic-Name-SNMP-custom | Check traffic of an network interface                           | X          |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-symbol-wing-snmp.md:34]:
```suggestion
| Traffic-Generic-Name | Net-Symbol-Wing-Traffic-Generic-Name-SNMP-custom | Check traffic of a network interface                           | X          |
```

File: pp/integrations/plugin-packs/procedures/network-symbol-wing-snmp.md [hunk 6]
```diff
@@ -0,0 +1,551 @@
...
+| Traffic-Global       | Net-Symbol-Wing-Traffic-Global-SNMP-custom       | Check traffic of an network interface                           |            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-symbol-wing-snmp.md:35]:
```suggestion
| Traffic-Global       | Net-Symbol-Wing-Traffic-Global-SNMP-custom       | Check traffic of a network interface                           |            |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by lucie-dubrunfaut — APPROVED** [review]

**Pull request merged**